<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGifMaker</class>
 <widget class="QDialog" name="DialogGifMaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save to Gif</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxWidth">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxDelay">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Delay</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxRescale">
       <property name="text">
        <string>Rescale by width</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Number of frames</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="labelNumFrames">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>pixel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">padding-left:20;padding-right:20</string>
     </property>
     <property name="text">
      <string>Click Add to add the current view to the GIF sequence. Click Save As to finish and save the gif.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave_2">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxRescale</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBoxWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAdd</sender>
   <signal>clicked()</signal>
   <receiver>DialogGifMaker</receiver>
   <slot>OnButtonAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>DialogGifMaker</receiver>
   <slot>OnButtonClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>DialogGifMaker</receiver>
   <slot>OnButtonSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave_2</sender>
   <signal>clicked()</signal>
   <receiver>DialogGifMaker</receiver>
   <slot>OnButtonClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnButtonAdd()</slot>
  <slot>OnButtonClear()</slot>
  <slot>OnButtonSave()</slot>
  <slot>OnButtonClose()</slot>
 </slots>
</ui>
